Sources/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>NSPrivacyTracking</key>
+	<false/>
+	<key>NSPrivacyCollectedDataTypes</key>
+	<array>
+		<dict>
+			<key>NSPrivacyCollectedDataType</key>
```

Comment:
According to the [docs](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files), the value for this key should be an array of dictionaries.

> For each type of data your app or third-party SDK collects, add a dictionary to the NSPrivacyCollectedDataTypes array in your privacy information file. Add the following keys to the dictionary.
>
> `NSPrivacyCollectedDataType`
> A string that identifies the type of data your app or third-party SDK collects. Choose the value from the list of data types below that matches the data your app or third-party SDK collects.
> 
> `NSPrivacyCollectedDataTypeLinked`
> A Boolean that indicates whether your app or third-party SDK links this data type to the user’s identity. For more information, see Data linked to the user in [App privacy details on the App Store](https://developer.apple.com/app-store/app-privacy-details/#linked-data).
> 
> `NSPrivacyCollectedDataTypeTracking`
> A Boolean that indicates whether your app or third-party SDK uses this data type to track.
> 
> `NSPrivacyCollectedDataTypePurposes`
> An array of strings that lists the reasons your app or third-party SDK collects the data. Choose values from the list of purposes below that match the reasons your app or third-party SDK collects this data type.

It's just like the privacy nutrition label in the App Store, in that regard.

Reply:
I'm confused. This is already the same as the docs describe right?

Reply:
Yes. Now that I'm looking at it again, it's the same as in the docs